source/macos/AdaptiveCards/AdaptiveCards/Renderers/InputNumberRenderer.swift

```diff
@@ -141,18 +155,25 @@ open class ACRNumericTextField: NSView, NSTextFieldDelegate {
     }
     
     private func setupViews() {
-        addSubview(textField)
-        addSubview(stepper)
+        addSubview(contentStackView)
+        contentStackView.addArrangedSubview(contentView)
+        contentView.addSubview(textField)
+        contentView.addSubview(stepper)
     }
     
     private func setupConstraints() {
-        textField.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
-        textField.topAnchor.constraint(equalTo: topAnchor).isActive = true
-        textField.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
+        contentStackView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
```

Comment:
can we add a helper functions for these **contentStackView.toFill(view: view)**, like we have in uc client for filling the whole view.

Reply:
lets connect for this.